TrkFilters/src/HelixFilter_module.cc

```diff
@@ -62,6 +62,8 @@ namespace mu2e
         fhicl::Atom<int>                debugLevel           {     Name("debugLevel"),              Comment("debugLevel")     , 0 };
         fhicl::OptionalAtom<bool>       prescaleUsingD0Phi   {     Name("prescaleUsingD0Phi"),      Comment("prescaleUsingD0Phi") };
         fhicl::Table<PhiPrescalingParams::Config>             prescalerPar{     Name("prescalerPar"),      Comment("prescalerPar") };
+        fhicl::Atom<int>                noFilter             {     Name("noFilter"),                Comment("Don't filter anything"),0 };
```

Comment:
If the goal of this parameter is to allow bypassing the filter while keeping sequences the same, the standard way to do that is to replace the filter with that label with an instance of FixedFilter: https://github.com/Mu2e/Offline/blob/main/Filters/src/FixedFilter_module.cc
Then you don't need noFilter as an option in every module.

Reply:
What is the use case for enabling noFilter? 

This module will create a TriggerInfo object that says the event failed when it in fact it failed but was passed because of noFilter.  This feels dangerous to me - I would like our event information to be internally self consistent.      
For that reason I prefer Dave's recommendation for the use cases I can think of.

Reply:
And the some comment in the other places this comes up.

Reply:
I don't think for trigger studies the proposed solution is good. When creating ntuples for trigger optimization we want to look at what passed and what was rejected using the same file, thus we run the filter and store the trigger results (the list of the pointers that passed the selection).

Reply:
Just to be clear - I think you are saying that the TriggerInfo for failed events is important for this use case - that's what you want to put in the ntuple, right?

Can you achieve the same effect using the prescale feature?  (Perhaps the implementation of the prescale will need maintenance to make tihs work?).